src/authorize.js

```diff
@@ -95,35 +104,28 @@ async function authorize(token, params, referer) {
 
   const id = params.id.split("/").slice(-1)[0];
 
-  const currentUser = getCurrentUser(token);
-  const doc = await getDoc(id);
-  const visibility = getVisibility(doc._source);
-  const username = isObject(currentUser) ? currentUser.displayName[0] : 'Anonymous';
-
-  console.log('User:', username);
-  console.log('Item ID:', id);
-  console.log('Visibility:', visibility);
+  return await getImageAuthorization(id);
+}
 
-  switch(visibility) {
-    case 'open':          return true;
-    case 'authenticated': return isObject(currentUser);
-    case 'restricted':    return false;
-    default:              return true;
-  }
+async function getImageAuthorization(id) {
+  const response = await axios.get(
+    `${dcApiEndpoint}/file-sets/${id}/authorization`
+  );
```

Comment:
Since the IIIF auth function is taking an incoming request and then turning around and asking for authorization from the API, it also needs to pass the incoming request's cookies through to the API by pulling the header out of the event and including it in this call. Otherwise, the API will think every request from the IIIF server is an anonymous request.

(The API token cookie will be part of the request, because it's pinned to the `.library.northwestern.edu` domain, which the IIIF server is part of.)